website/docs/docs/deploy/jobs.md

```diff
@@ -1,23 +1,24 @@
 ---
 title: "Jobs in dbt Cloud"
 sidebar_label: "About Jobs"
-description: "Learn about deploy jobs and continuous integration (CI) jobs in dbt Cloud and what their differences are." 
+description: "Learn about the different job types in dbt Cloud and what their differences are." 
 tags: [scheduler]
 pagination_next: "docs/deploy/deploy-jobs"
 ---
 
-In dbt Cloud, there are two types of jobs: 
+These are the available job types in dbt Cloud: 
 - [Deploy jobs](/docs/deploy/deploy-jobs) &mdash; To create and set up triggers for building production data assets
 - [Continuous integration (CI) jobs](/docs/deploy/continuous-integration) &mdash; To create and set up triggers for checking code changes
+- [Merge jobs](/docs/deploy/merge-jobs) &mdash; To create and set up triggers for merged pull requests
 
-Below is a comparison table that describes how deploy jobs and CI jobs behave:
+Below is a comparison table that describes the behaviors of the different job types:
 
-|  | Deploy Jobs | CI Jobs |
-| --- | --- | --- |
-| Purpose | Builds production data assets. | Builds and tests new code before merging changes into production. |
-| Trigger types | Triggered by a schedule or by API. | Triggered by a commit to a PR or by API. |
-| Destination | Builds into a production database and schema. | Builds into a staging database and ephemeral schema, lived for the lifetime of the PR. |
-| Execution mode | Runs execute sequentially, so as to not have collisions on the underlying DAG. | Runs execute in parallel to promote team velocity. |
-| Efficiency run savings | Detects over-scheduled jobs and cancels unnecessary runs to avoid queue clog. | Cancels existing runs when a newer commit is pushed to avoid redundant work. |
-| State comparison | Only sometimes needs to detect state. | Almost always needs to compare state against the production environment to build on modified code and its dependents. |
-| Job run duration | Limit is 24 hours. | Limit is 24 hours. |
\ No newline at end of file
+|  | Deploy jobs | CI jobs | Merge jobs |  
```

Comment:
I don't have any great suggestions but this table is a little hard to parse at first because the job types are along the top. I see why we've done that (it would look silly to have 7 columns and 3 rows.)

Maybe just bolding the first column would help?